src/style.css

```diff
@@ -0,0 +1,39 @@
+* {
+  margin: 0;
```

Comment:
Don't reset margins using * selector. It applies to all the elements, which is not good for performance